dcm4che-imageio-opencv/src/main/java/org/dcm4che3/opencv/StreamSegment.java

```diff
@@ -116,14 +116,8 @@ public static boolean supportsInputStream(Object iis) {
 
     private static StreamSegment getFileStreamSegment(SegmentedInputImageStream iis) {
```

Comment:
These changes simplify the access to the segments and allow to limit the "add export" retlated to the reflection (mandatory since Java 16).